core/src/test/java/com/sequenceiq/cloudbreak/core/bootstrap/service/ClusterBootstrapperTest.java

```diff
@@ -111,6 +108,7 @@ public void shouldUseReachableInstances() throws CloudbreakException, Cloudbreak
         Cluster cluster = new Cluster();
         cluster.setGateway(new Gateway());
         when(stack.getCluster()).thenReturn(cluster);
+        GatewayConfig gatewayConfig = new GatewayConfig("host1", "1.1.1.1", "1.1.1.1", 22, "i-1839", false);
```

Comment:
Was this an incidental change?

Reply:
No it fixes an NPE.  The gateway config was a mock, the logic i added needed to compare ip's so it being null throws a NPE.